indexer-api/src/infra/storage/postgres/unshielded_utxo.rs

```diff
@@ -50,6 +50,7 @@ impl UnshieldedUtxoStorage for PostgresStorage {
                 creating_transaction_id, spending_transaction_id
             FROM unshielded_utxos
             WHERE creating_transaction_id = $1
+            ORDER BY output_index ASC
```

Comment:
In all the other ORDER BY clauses we do not use an explicit ASC. Please drop it here and below.